### PR TITLE
Improve top-level command descriptions; add Handbook links

### PR DESCRIPTION
**Pull request by hearvox**
Related: [/wp-cli/issues/4302](https://github.com/wp-cli/wp-cli/issues/4302)

Changes:
___
Improve 'wp cap' command description in:
https://github.com/wp-cli/role-command/blob/master/src/Capabilities_Command.php
Command: wp cap
Current description: Manage user capabilities.

New description:
Adds, removes, and lists capabilities of a user role.

See references for [Roles and Capabilities](https://codex.wordpress.org/Roles_and_Capabilities)and [WP User class](https://codex.wordpress.org/Class_Reference/WP_User).

___
Improve command description in:
https://github.com/wp-cli/role-command/blob/master/src/Role_Command.php
Command: wp role
Current description: Manage user roles.

New description:
Manages user roles, including creating new roles and resetting to defaults.

See references for [Roles and Capabilities](https://codex.wordpress.org/Roles_and_Capabilities)and [WP User class](https://codex.wordpress.org/Class_Reference/WP_User).

### CODE REVIEW

File: src/Capabilities_Command.php [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <?php
 
 /**
- * Manage user capabilities.
+ * Adds, removes, and lists capabilities of a user role.
+ *
+ * See references for [Roles and Capabilities](https://codex.wordpress.org/Roles_and_Capabilities)and [WP User class](https://codex.wordpress.org/Class_Reference/WP_User).
```
Comment (miya0001) [inline src/Capabilities_Command.php]: Can you add a space before `and`?

File: src/Role_Command.php [hunk 2]
```diff
@@ -1,7 +1,9 @@
 <?php
 
 /**
- * Manage user roles.
+ * Manages user roles, including creating new roles and resetting to defaults.
+ *
+ * See references for [Roles and Capabilities](https://codex.wordpress.org/Roles_and_Capabilities)and [WP User class](https://codex.wordpress.org/Class_Reference/WP_User).
```
Comment (miya0001) [inline src/Role_Command.php]: Can you add a space before `and`?

**Comment by miya0001** [comment]
`wp role`:



`wp cap`:

**Comment by miya0001** [comment]
I feel uncomfortable at the position of the line break at description.
But I can understand it is difficult to solve it.

@wp-cli/committers Any thoughts?

File: src/Role_Command.php (hunk 2 above)
Comment (hearvox) [reply src/Role_Command.php]: Will do, then I'll close this PR and submit a new one, correct. Before I do I'll wait to see what folk say about the line-break.

Long and multiple sentences in the descriptions are a problem b/c forced line breaks look OK in the code, for example, of this existing description:
https://github.com/hearvox/shell-command/edit/master/src/Shell_Command.php

But are grammatically wrong, bad for a11y, and look funny in the Handbook:
https://developer.wordpress.org/cli/commands/shell/

**Comment by schlessera** [comment]
Authoring props: @hearvox & @ako80218

**Comment by schlessera** [comment]
@miya0001 We'll deal with the word-wrapping through the parsing code.

**Pull request merged**